src/lib/config/project-config.ts

```diff
@@ -6,22 +6,29 @@ import { RootConfig, Config, InvalidConfigError } from ".";
 // Represents `.fauna-project`
 export class ProjectConfig {
   defaultStack?: string;
+  fslDir?: string;
   stacks: { [key: string]: Stack };
 
   /**
    * This method is used to obtain an empty project config when fauna project init is used.
    */
-  static emptyConfig(): ProjectConfig {
-    return new ProjectConfig({});
+  static initialConfig(fslDir?: string): ProjectConfig {
+    return new ProjectConfig({}, undefined, fslDir);
   }
 
-  private constructor(stacks: { [key: string]: Stack }, defaultStack?: string) {
+  private constructor(
+    stacks: { [key: string]: Stack },
+    defaultStack?: string,
+    fslDir?: string
+  ) {
     this.stacks = stacks;
     this.defaultStack = defaultStack;
+    this.fslDir = fslDir;
   }
 
   static fromConfig(config: Config): ProjectConfig {
     const defaultStack = config.strOpt("default");
+    const fslDir = config.strOpt("fsl_directory");
```

Comment:
I think this should be called `schema_directory`. We call it schema when referring to it by name elsewhere, FSL is just the file extension.